app/utilities/decimal_places.py

```diff
@@ -1,11 +1,11 @@
 from decimal import Decimal
 
 import flask_babel
-from babel import Locale, numbers, units
+from babel import Locale, numbers
 
 
 def unit_dec(value: int | Decimal | float):
-    decimal_places = len(str(value).split(".")[-1])
+    decimal_places = len(str(value).split(".")[1])
```

Comment:
if value is an `int` this is going to break - you'll need to check it has decimal places first
an `isinstance` check won't be enough, as you could have something like `Decimal("222")`

There are functions for decimals that can get the number of places regardless of if its whole or not like `abs(Decimal("2.00").as_tuple().exponent)` but I quite like the simplicity of len, so probably just need to check the string has a `.`

Reply:
We could use the `get_decimal_precision` method to do this. Happy with either.

Reply:
`get_decimal_precision` won't account for trailing zeroes, otherwise I'd agree

Reply:
Ah fair point.